rasa/core/agent.py

```diff
@@ -228,7 +228,7 @@ async def load_agent(
         lock_store = LockStore.create(endpoints.lock_store)
         generator = endpoints.nlg
         action_endpoint = endpoints.action
-        model_server = endpoints.model if endpoints.model else model_server
+        model_server = endpoints.model or model_server
```

Comment:
Function `load_agent` refactored with the following changes:

- Simplify if expression by using or ([or-if-exp-identity](https://docs.sourcery.ai/refactorings/or-if-exp-identity/))